crates/codegen/parser_templates/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "parser_templates"
```

Comment:
nit: to match how other crates are defined.

```suggestion
name = "codegen_parser_templates"
```

Reply:
never used, but ok

Reply:
Not blocking, but the standardization makes it easier to grep across all file types.

Reply:
done